src/components/LightSwitch.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
+import { useTheme } from 'next-themes';
+import cx from 'classnames';
+
+export const LightSwitch = () => {
+  const [mounted, setMounted] = useState(false);
+  const { theme, setTheme } = useTheme();
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
```

Comment:
Why was it needed? Was it switching between dark and light?

Reply:
<img width="676" alt="image" src="https://github.com/GLObus303/sportujspolu/assets/47262567/17a301b4-c3ce-4573-bb36-2870c1456903">
 to fix this

Reply:
On the server is light and client dark

Reply:
It probably uses some window/browser API unavailable on the server.  Not sure I like it

Reply:
Removed: ```
  useEffect(() => {
    setMounted(true);
  }, []);
```